cresana/bfield.py

```diff
@@ -62,6 +67,13 @@ def evaluate_B(self, pos, derivatives=False):
 
         Bz[...] = C/(2*alpha**2*beta)*((a**2 - r**2)*E_k2 + alpha**2*K_k2)
 
```

Comment:
Could gate the new raw arrays by `if self.alpha!=0:` Saves some time and memory. Unfortunately, CRESana has a big memory footprint already if it is run with a long trajectory. If I recall correctly it does not directly matter with the current setups that I use because the B_field evaluation is done with the Interpolation function. Without the interpolation function the line `B_raw=np.empty_like(pos)` will create a new array of size `n_samples*n_antennas` during the simulation.